BankexWallet/InterfaceElements/DesignElements/Onboarding/CircleView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  CircleView.swift
+//  BankexWallet
+//
+//  Created by Георгий Фесенко on 01/06/2018.
+//  Copyright © 2018 Alexander Vlasov. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class CircleView: UIView {
+
+    
+    @IBInspectable var filled: Bool = true
+    @IBInspectable var strokeWidth: CGFloat = 2.7
+    @IBInspectable var strokeColor: UIColor = #colorLiteral(red: 0.7568627451, green: 0.7921568627, blue: 0.8235294118, alpha: 1)
```

Comment:
There's enum with colors. Maybe it's worth to move these there?

Reply:
fixed